Description: Fix libextractor name in manpage
 Replace the display name of GNU libextractor with the name of the manpage
 in the whatis entry.
Author: Bertrand Marc <beberking@gmail.com>
Applied-Upstream: commit:r27537
--- a/doc/libextractor.3
+++ b/doc/libextractor.3
@@ -1,6 +1,6 @@
 .TH LIBEXTRACTOR 3 "Sept 4, 2012" "GNU libextractor 1.0.0"
 .SH NAME
-GNU libextractor \- meta\-information extraction library 1.0.0
+libextractor \- meta\-information extraction library 1.0.0
 .SH SYNOPSIS
 
 \fB#include <extractor.h>
